File: app/src/main/java/com/app/activeparks/data/model/authorisation/Payload.java
package com.app.activeparks.data.model.authorisation;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Payload {

    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("permissions")
    @Expose
    private List<String> permissions = null;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public List<String> getPermissions() {
        return permissions;
    }

    public void setPermissions(List<String> permissions) {
        this.permissions = permissions;
    }

}

File: app/src/main/java/com/app/activeparks/data/model/registration/Payload.kt
package com.app.activeparks.data.model.registration

import com.google.gson.annotations.SerializedName

data class Payload(

	@SerializedName("subRole")
	val subRole: Boolean,
	@SerializedName("id")
	val id: String,
	@SerializedName("sessionId")
	val sessionId: String
)